### DISCUSSION TITLE
Why there are only small portion of the Channels available?

### DISCUSSION THREAD

**Discussion by blufish1234** [xspanger3770/GlobalQuake#293] [Q&A] [answered]
![image](https://github.com/xspanger3770/GlobalQuake/assets/97332474/5159b904-e284-4c7c-9032-f4743d17709d)
I don't know if this happens to you guys but, why are there so many unavailable channels?

**Comment by YacineBoussoufa** [comment, marked as answer]
The FDSNWS (containing station location information) knows the location of some stations but there isn't a public Seedlink Server containing the actual waveform data, therefore they appear as unavailable. For more stations you could connect to our server at launch-up at server.globalquake.net port 38000

**Reply by blufish1234** [reply]
Thanks!
